lib/databases/postgres/worker/fee-switch.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+package worker
+
+import (
+	"fmt"
+	"database/sql"
+
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/types/ethereum"
+	"github.com/fluidity-money/fluidity-app/lib/postgres"
+	"github.com/fluidity-money/fluidity-app/lib/types/network"
+)
+
+func GetFeeSwitch(originalAddress ethereum.Address, network_ network.BlockchainNetwork) *FeeSwitch {
+	postgresClient := postgres.Client()
+
+	statementText := fmt.Sprintf(`
+		SELECT new_address
+		FROM %s
+		WHERE network = $1 AND original_address = $2`,
```

Comment:
missing a `;`

Reply:
I don't think this needs semicolons here!